grails-app/controllers/au/org/ala/collectory/DataController.groovy

```diff
@@ -280,7 +280,7 @@ class DataController {
 
     @Path("/ws/{entity}/{uid}")
     @Produces("application/json")
-    @RequireApiKey(roles = ["ROLE_EDITOR"])
+    @AlaSecured(value = ['ROLE_EDITOR'], anyRole = true)
```

Comment:
Update to use the configured ROLE_EDITOR instead of a static string. Also permit m2m with admin scope. See CollectoryWebServicesInterceptor.